apps/emqx_resource/src/emqx_resource_manager.erl

```diff
@@ -229,14 +231,25 @@ set_resource_status_connecting(ResId) ->
 -spec lookup(resource_id()) -> {ok, resource_group(), resource_data()} | {error, not_found}.
 lookup(ResId) ->
     case safe_call(ResId, lookup, ?T_LOOKUP) of
-        {error, timeout} -> ets_lookup(ResId);
+        {error, timeout} -> lookup_cached(ResId, [metrics]);
```

Comment:
Well, no, because the PR you mentioned made them disappear only on the API level, where responses are being formatted. This change only makes sure that `lookup/1` semantics become more consistent (e.g. _always return metrics info_ instead _sometimes return metrics info_). Ideally I think we'd want to stop asking the backend for metrics when we don't want them, but that'll require few more changes, and smells like a followup PR material.

Reply:
I see.  I thought that at this point we wouldn't want metrics since they now have their own endpoint where they're collected explicitly.

Reply:
That's right. I attempted to make a commit which optimizes that out, but:
1. It became quite involved.
2. Only then I noticed that #10114 haven't been merged yet. 🫠

So I guess I'll open a followup PR once it's merged.